### PR TITLE
MNT Fix broken constraint on .travis.yml

### PR DESCRIPTION
**Pull request by maxime-rainville**
That constraint is broken and [breaks the build](https://travis-ci.com/github/silverstripe/silverstripe-elemental-bannerblock/jobs/478036094). 

I've made it generic so it can install whatever is greater than or equal to 4.7

### CODE REVIEW

**Comment by dhensby** [comment]
Builds still failing, looks like this equivalent fix is needed (to `.travils.yml`) https://github.com/silverstripe/silverstripe-elemental-bannerblock/pull/67

**Comment by maxime-rainville** [comment]
Tried quoting the constraint. Let's see what travis think about that.

**Review by emteknetnz — CHANGES_REQUESTED** [review]
REQUIRE_RECIPE should be `"4.7.x-dev || "` - it's the [documented way](https://github.com/silverstripe/silverstripe-travis-shared#use-a-range-provsion-and-define-a-minimum-recipe-version-to-match-a-minimum-version-in-composerjson) to require a minimum version of a recipe

The installation issue is only with REQUIRE_EXTRA

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ import:
 env:
   global:
     # Require at least recipe 4.7.x-dev to satisfy the silverstripe/admin:1.7@dev requirement
-    - REQUIRE_RECIPE="4.7.x-dev || "
+    - REQUIRE_RECIPE="^4.7"
```
Comment (emteknetnz) [inline .travis.yml:9]: This should stay as `"4.7.x-dev || "`

**Comment by emteknetnz** [comment]
Build has been failing on this module for 4 months.  Likely it started after [this commit](https://github.com/silverstripe/silverstripe-travis-shared/commit/92913d736b3a9d40e61d42fdb533cf60f679d2ef) in travis-shared

We DO NOT want to change the quoting in travis-shared - I've had multiple issues before with this and the way it's done in travis-shared `$REQUIRE_EXTRA` instead of `"$REQUIRE_EXTRA"` is that way for a reason

Looks like the `\"` style of quoting didn't work.  Might be worth trying single quotes i.e. `REQUIRE_EXTRA='dnadesign/silverstripe-elemental:"^4.5@dev || 4.x-dev"'`

**Comment by emteknetnz** [comment]
Actually just saw Dan's comment about it being equivalent of https://github.com/silverstripe/silverstripe-elemental-bannerblock/pull/67 which is passing (expect for PHP8, which is pending elemental php8 support) - we should use that PR instead

**Pull request closed without merging**